test/Microsoft.AspNet.Mvc.Razor.Test/RazorViewTest.cs

```diff
@@ -555,11 +555,13 @@ public class RazorViewTest
                     await v.RenderSectionAsync("sectionA");
                 });
             });
+            nestedLayout.Path = "NestedLayout";
```

Comment:
So did these start failing because the empty string was detected as a circular reference?